CMSgov/eAPD
# fixed file that was overwritten with older version

### Description
a PR overwrote the previous fix for only expiring admins, adding this code back in to restore that

### Steps to manually verify this change

1. login as a user that is not a state admin
2. verify that their affiliation is not revoked

### This pull request is ready to review when


### This pull request can be merged when
